lib/writer.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const { Readable } = require('stream');
+const { identifyCssModule, bundleCssModule } = require('./util');
+const { existsSync } = require('fs');
+const { isAbsolute } = require('path');
+const assert = require('assert');
+
+module.exports = class Writer extends Readable {
+    constructor (files = []) {
+        super({ objectMode: true });
+
+        assert(
+            Array.isArray(files) || typeof files === 'string',
+            `Expected 'files' to be of type 'Array' or a 'string', instead got '${typeof this.files}'`
```

Comment:
`typeof files`, `this.files` is undefined at this point

Reply:
baam, thats what I get for not asserting on the error message

Reply:
you can to `toThrowErrorMatchingSnapshot()` if you're feeling hipster (it checks the message, not the stack)

Reply:
ah nice, will look into it. Need to increase my Jest knowledge before Thursday so good to check such things out.

Reply:
Feel free to fire questions at me, I'm a collaborator on jest